### DISCUSSION TITLE
How to import the mesh in PyFluent?

### DISCUSSION THREAD

**Discussion by MayconSilva7** [ansys/pyfluent#1881] [General]
Hello, 

My goal is import a fluent mesh and after set the boundary conditions. I mean, open the Fluent Launcher, read the mesh file and after change to solver. I don't want to change anything in the mesh.
 
I've been trying to find a function to import the mesh, but there are just to case and data.

```
#Import library
import ansys.fluent.core as pyfluent
from ansys.fluent.core import examples
from ansys.fluent.core import launch_fluent


meshing_session = pyfluent.launch_fluent(mode="meshing")
meshing_session.tui.file.read_case("mesh.h5")
```

Basically, I would like to open the Fluent with generic conditions. And then create a case using the PyFluent. Because, if I just open Fluent, all the options are disabled.

Thank you very much in advance.

**Comment by mkundu1** [comment]
If you want to read the mesh in solver mode, the following should work:
```
session = pyfluent.launch_fluent()
session.tui.file.read_case("xyz.msh.h5")
```
You can also use settings API to read the mesh in solver mode
```
solver.file.read_mesh(file_name="xyz.msh.h5")
```

**Reply by MayconSilva7** [reply]
Resolved. Thank you very much!
